app/api/helpers/scheduled_jobs.py

```diff
@@ -87,7 +87,7 @@ def send_event_fee_notification():
             fee_total = 0
             for order in orders:
                 for order_ticket in order.tickets:
-                    ticket = safe_query(db, Ticket, 'id', order_ticket.ticket_id, 'ticket_id')
+                    ticket = safe_query(db, Ticket, 'id', order_ticket.id, 'id')
```

Comment:
Why even do this? We already have the ticket as order_ticket?

Reply:
@iamareebjamal - *safe query*.

Reply:
@iamareebjamal Done the change

Reply:
@codedsun If someone added a ticket in order when it was not deleted, and then placed an order and now the ticket was deleted. Should it add the fee of deleted ticket in the order invoice or not?

Reply:
Yes, cool! 👍 got that